src/posts/2020-01-27-stencil-getting-started.md

```diff
@@ -0,0 +1,70 @@
+---
+title: 'Aan de slag met Stencil'
+tags: Stencil, Web Components, Design System
+comments: true
+author: danny
+min_read: 4
+date: 2020-01-27
+---
+
+### Wat is Stencil?
+[Stencil](https://stenciljs.com/) is een compiler die Web Components (Custom Elements v1) genereert. Stencil combineert de beste concepten van de meest populaire frameworks in een simpele tool.
+
+Stencil maakt gebruik van features zoals:
+* Virtual DOM
+* Async rendering
+* Reactive data-binding
+* TypeScript
+* JSX
+
+Stencil is ontwikkeld door het Ionic framework team.
+
+### Browser Support
+
+Stencil bouwt Web Components die te gebruiken zijn in de populaire desktop en mobile browsers. Voor de browsers die geen ondersteuning bieden voor Custom Elements v1 (De API waarop Web Components zijn gebaseerd) maakt Stencil gebruik van polyfills.
+Stencil maakt hiervoor gebruik van een dynamic loader die de polyfill alleen inlaadt in browsers die het nodig zijn.
+
+![verbindingen](assets/stencil-getting-started/stencil-browsersupport.png)
+
+### Getting Started
+
+Stencil vereist een recente LTS versie van NodeJS en npm (npm 6 of hoger). Zorg er dus voor dat deze geïnstalleerd of up to date zijn voordat je verdergaat.
+
+```
+ npm init stencil
```

Comment:
Zou er ook niet een stap aan vooraf moeten gaan  on stencil te installeren? Of is dit het startpunt?

Reply:
Dit is het daadwerkelijke start punt, je kiest dan een van de opties van wat jij wilt als project, in dit geval **component** en na het initialiseren moet je de stap bij "Updating Stencil" uitvoeren daarmee installeer je dan daadwerkelijk de laatste versie van Stencil. Ik zal dit nog even duidelijker verwoorden want dit is inderdaad een beetje onduidelijk.

Reply:
Ok nice!